api/config/rolepermission.php

```diff
@@ -0,0 +1,521 @@
+<?php
+
+return [
+    /*
+    |--------------------------------------------------------------------------
+    | Actions
+    |--------------------------------------------------------------------------
+    |
+    | A map of all actions available within the application.
+    |
+    */
+    'actions' => [
```

Comment:
Would it not make sense to use the same number of characters for each key for consistency? Maybe at least two characters in case another action is added in the future.

Reply:
I'm going to take the [suggestion to use full names](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5560#discussion_r1093699187) here.